### ISSUE TITLE
Typo on runner package docstring

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5740]
|<img alt="tyteen4a03's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| tyteen4a03 reported|
|-|-|
|Trac ID|trac#5740|
|Type|defect|
|Created|2012-07-08 09:55:48Z|

Should be Twisted runner instead of runer.

Attachments:

* [patch.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/d19/d19294bfbf103f42537c59ea6b610f317a410776/c9d1be3a12c00667bfdaa35aaaabbe1d8fdcd585.patch) (316 bytes) - added by tyteen4a03 on 2012-07-08 09:56:29Z - Patch file

<details><summary>Searchable metadata</summary>

```
trac-id__5740 5740
type__defect defect
reporter__tyteen4a03 tyteen4a03
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__runner runner
keywords__runner_typo runner typo
time__1341741348000000 1341741348000000
changetime__1358910115000000 1358910115000000
version__None None
owner__thijs thijs
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

There's also a copyright header missing in that file. I'll apply your patch and merge that, thanks.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [36892]) Fix typo in the docstring of the Twisted runner package.

Author: tyteen4a03
Reviewer: thijs
Fixes: [#5740](https://github.com/twisted/twisted/issues/5740)
